app/assets/stylesheets/pages/_sign-in.scss

```diff
@@ -0,0 +1,50 @@
+.container {
```

Comment:
`container` is an important bootstrap class. Please don't target it. You can change the name of your custom class.